src/compliment/sources/class_members.clj

```diff
@@ -209,43 +274,67 @@
 ;; ## Static members
 
 (defn static-member-symbol?
-  "Tests if prefix looks like a static member symbol, returns parsed parts."
-  [x]
+  "Tests if `x` looks like a static member symbol, returns parsed parts."
+  [^String x]
```

Comment:
This is noise. Please don't touch if you don't change it meaningfully or make it better.